### DISCUSSION TITLE
Date of birth of child - facial tagging before being born

### DISCUSSION THREAD

**Discussion by dvpfig** [immich-app/immich#7605] [Q&A]
Hi,

I added date of birth to my 5 year old son. 

The facial recognition is quite okay for babies but as expected it has many misses or failed tagging. 

However, I was surprised to see that it tags pictures of my son with dates before he is born. 

A simple rule for face tagging could be added to avoid tagging a person to a picture dated before birth date (if that information in available). 

I think more than this simple rule becomes quite hard to try to adjust facial recognition with expected age at the date of picture.

But such simple rule," no tags before born " makes sense, right? 

Is there any configuration that one needs to do to adjust facial recognition to take into account the age or date of birth?

**Comment by aviv926** [comment]
Duplicate of #4743
